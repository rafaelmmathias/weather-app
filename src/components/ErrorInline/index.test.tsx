import { ErrorInline } from ".";
import renderer from "react-test-renderer";

describe("Component - ErrorInline", () => {
  it("should render the title and message", () => {
    const tree = renderer
      .create(
        <ErrorInline title="Error inline title" message="message error" />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
