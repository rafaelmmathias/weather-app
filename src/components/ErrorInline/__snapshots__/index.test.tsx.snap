// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - ErrorInline should render the title and message 1`] = `
<div
  className="sc-bczRLJ eFUIdW"
>
  <div
    className="sc-bczRLJ bpeZoH sc-gsnTZi bzWLFp"
  >
    <div
      className="sc-bczRLJ cDQXru"
    >
      <h1
        className="sc-dkzDqf hPtMBP"
      >
        Error inline title
      </h1>
    </div>
    <div
      className="sc-bczRLJ eFUIdW"
    >
      <p
        className="sc-eCYdqJ fhRWxa"
        color="#fff"
      >
        message error
      </p>
    </div>
  </div>
</div>
`;
